plugins/language_json.lua

```diff
@@ -0,0 +1,28 @@
+-- mod-version:3 priority:90
+
+local syntax = require "core.syntax"
+
+syntax.add {
+  name = "JSON",
+  files = { "%.json$" },
+  comment = nil,
+  patterns = {
+
+    { pattern = '\"[a-zA-Z0-9_]*\":', type = "keyword" }, -- key (string)
+    { pattern = '[0-9_]*:', type = "keyword" }, -- key (int)
+    { pattern = { '"', '"', "\\"}, type = "string" }, -- value
```

Comment:
I wonder if this should be adapted to disallow multiline strings.

Reply:
I've got it to ignore strings like:
```
"foo": "

bar",
```

Should it also ignore `"""`, `''` and the like?